### PR TITLE
WC: Navigation component

### PR DESCRIPTION
**Pull request by Prabhav-Sharma**
Made 3 variants of Navigations
-In row
-In columns with space-between
-Nav with hamburger button

### CODE REVIEW

File: Documentation/Components/Navigation/nav.html [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Navigation</title>
+    <link rel="stylesheet" href="nav.css">
+</head>
+<body>
+    <nav class="nav-row flex-row">
+        <div class="nav-logo flex-row">
+            <img class="nav-img" src="/images/helmet.png">
```
Comment (devananth) [inline Documentation/Components/Navigation/nav.html]: `alt` attribute is missing in `img` tag

File: Documentation/Components/Navigation/nav.html [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+        <ul class="nav-links flex-row">
+            <li><a class="nav-link" href="#">Home</a></li>
+            <li><a class="nav-link" href="#">Docs</a></li>
+            <li><a class="nav-link" href="#">GitHub</a></li>
+        </ul>
+    </nav>
+    <br>
+    <nav class="nav-column flex-column">
+        <div class="nav-logo flex-row">
+            <img class="nav-img" src="/images/helmet.png">
```
Comment (devananth) [inline Documentation/Components/Navigation/nav.html]: alt attribute is missing in img tag

**Review by devananth — COMMENTED** [review]
Everything else is fine

**Review by tewarimohit — APPROVED** [review]
Good to merge from my side..

File: Documentation/Components/Navigation/nav.html (hunk 2 above)
Comment (Prabhav-Sharma) [reply Documentation/Components/Navigation/nav.html]: I've added it now, thank you

**Pull request merged**